.github/workflows/release.yml

```diff
@@ -0,0 +1,26 @@
+name: Release
+on:
+  push:
+    branches: [main, development]
```

Comment:
This indicates that when we merge to either main or development the release workflow is triggered. Don't we just want this to be `main` ? @arijitAD

Reply:
The job will be triggered but it won't trigger a release. I will fix this after this release is done.